src/framework/preferences/js/ArrowScrolling.js

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2017 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_3_0_0 = fluid_3_0_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    /************************************************************************************
+     * Scrolling Panel Prefs Editor:                                                    *
+     * This is a mixin grade to be applied to a fluid.prefs.prefsEditor type component. *
+     * Typically used for responsive small screen presentations of the separated panel  *
+     * to allow for scrolling by clicking on left/right arrows                          *
+     ************************************************************************************/
+
+    fluid.defaults("fluid.prefs.arrowScrolling", {
+        gradeNames: ["fluid.modelComponent"],
+        selectors: {
+            // panels: "", // should be supplied by the fluid.prefs.prefsEditor grade.
+            scrollContainer: ".flc-prefsEditor-scrollContainer"
+        },
+        model: {
+            // panelMaxIndex: null, // determined by the number of panels calculated after the onPrefsEditorMarkupReady event fired
+            // scrollToIndex: null, // the raw index set by translateToScroll, will be transformed to the panelIndex
+            panelIndex: 0
+        },
+        events: {
+            // beforeReset: null // should be provided by the fluid.prefs.prefsEditor grade
+        },
+        modelRelay: {
+            target: "panelIndex",
+            forward: {excludeSource: "init"},
+            singleTransform: {
+                type: "fluid.transforms.limitRange",
+                input: "{that}.model.scrollToIndex",
+                min: 0,
+                max: "{that}.model.panelMaxIndex"
+            }
+        },
+        modelListeners: {
+            "panelIndex": {
+                listener: "fluid.prefs.arrowScrolling.scrollToPanel",
+                args: ["{that}", "{change}.value"]
+            }
+        },
+        listeners: {
+            "onReady.windowResize": {
+                "this": window,
```

Comment:
These global listeners will pile up if the user creates several of these components, especially dynamically, and the framework doesn't have enough insight into DOM listeners to get rid of them for you. You should add an onDestroy listener for this component that unregisters it using removeEventListener.